Symfony/CS/Tests/Fixer/AbstractFixerTestBase.php

```diff
@@ -37,6 +37,30 @@ protected function getTestFile($filename = __FILE__)
         return $files[$filename];
     }
 
+    /**
+     * Tests if a fixer fixes a given string to match the expected result.
+     *
+     * It is used both if you want to test if something is fixed or if it
+     * is not touched by the fixer. It also makes sure that the expected
+     * output does not change when run through the fixer. That means that
+     * you do not need two test cases like
+     * [$expected] and
+     * [$expected, $input] (where $expected is the same in both cases)
+     * as the latter covers both of them.
+     *
+     * @param string              $expected The expected fixer output.
+     * @param string|null         $input    The fixer input, or null
+     *                                      if it should intentionally
+     *                                      be equal to the output.
+     * @param \SplFileInfo|null   $file     The file to fix, or null
+     *                                      if unneeded.
+     * @param FixerInterface|null $fixer    The fixer to be used, or null
+     *                                      if it should be inferred
+     *                                      from the test name.
+     *
+     * @throws \InvalidArgumentException If $expected and $input are equal.
```

Comment:
we do not use that annotation.
it would be usefull if it would be followed by all methods, but it does not